src/Graphics/Glyphs/GlyphGeom.cc

```diff
@@ -276,9 +276,12 @@ void GlyphGeom::generateCylinder(const Point& p1, const Point& p2, double radius
   if (num_strips < 0) num_strips = 20.0;
   double r1 = radius1 < 0 ? 1.0 : radius1;
   double r2 = radius2 < 0 ? 1.0 : radius2;
-
+  
   //generate triangles for the cylinders.
-  Vector n((p1 - p2).normal()), u = (10 * n + Vector(10, 10, 10)).normal();
+  Vector n((p1 - p2).normal());
+  Vector u = (10 * n + Vector(10, 10, 10)).normal();
+  if(n.x() == n.y() && n.x() == n.z())u = (10 * n + Vector(10, 0, 0)).normal();
```

Comment:
```suggestion
  if (n.x() == n.y() && n.x() == n.z())
    u = (10 * n + Vector(10, 0, 0)).normal();
```

Reply:
The code fixes @jessdtate's use case. First, let's improve the spacing.

Reply:
However, I'm concerned about the magic number `10` appearing twice, and the check comparing doubles exactly. We can discuss at today's meeting.